core/templates/dev/head/player/ConversationSkinDirective.js

```diff
@@ -247,11 +247,16 @@ oppia.animation('.conversation-skin-animate-card-contents', function() {
   };
 });
 
-oppia.directive('conversationSkin', [function() {
+oppia.directive('conversationSkin', ['urlService', function(urlService) {
   return {
     restrict: 'E',
     scope: {},
-    templateUrl: 'skins/Conversation',
+    link: function(scope) {
+      var isIframed = urlService.isIframed();
+      scope.myTemplate = isIframed ?
```

Comment:
This is nice, I hadn't actually realized you could do something like this. If it works then I think it's quite an elegant way of switching the templates.